metadata.json

```diff
@@ -12,13 +12,9 @@
     }
   ],
   "requirements": [
-    {
-      "name": "puppet",
-      "version_requirement": ">= 7.0.0 < 9.0.0"
-    },
     {
       "name": "openvox",
-      "version_requirement": ">= 7.0.0 < 9.0.0"
```

Comment:
I have an interest in supporting openvox-agent 7 because Satellite is still on 7 now. I'm willing to put in the work to make openvox 7 work in CI

Reply:
I'm fine with that, but someone needs to invest the time to publish the openvox 7 gem.